docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
-via the protocol itself.
+##### Liquidity Pool 
+2.  A  mSOL/SOL unstake pool - through which Sunrise becomes a liquidity provider to Marinade
+The liquidity pool is made up of SOL, mSOL and Liquidity Pool tokens
 
-### I am a validator, how can I get involved?
+  
+#### Step 3: Accruing yield
+###### Stake Pools
+At the end of each epoch (every 2 or 3 days), yield gets paid out into stake accounts, and the value of the overall stake pool increases. 
+As mSOL and bSOL represent a share of their respective stake pools, the value of mSol and bSOL also increases.
 
-In conjunction with the [Solana Foundation](https://solana.org/), we are exploring
-establishing a green stake pool, composed of validators that use renewable energy for
-their operations.
+The yield acrrued on the staked SOL, is equal to the value of mSOL and bSOL held by Sunrise minus the value of SOL staked (calculated as the circulatinbg supply of gSOL).
 
-Once the pool is established, Sunrise will direct stake to this pool, resulting in positive
-climate impact on both sides of the transaction:
+###### Liquidity Pool
 
-- Validators have an incentive to use renewable energy, reducing the carbon cost of the Solana network.
-- Stake yield is directed towards offsetting carbon emissions outside the solana network.
 
-### Where does the earned yield go?
+### Unstaking
+If you unstake your SOL, Sunrise calculates how much mSOL to sell back to Marinade to receive your SOL
+For example, if you decides to unstake 1 SOL, Sunrise will calculate how much mSOL to sell to Marinade (and bSOL to sell back to BlazeStake), in order to receive back that 1 SOL.
+* Once the latest update is made to send staked SOL to the Marinade Liquidity Pool, Sunrise will be selling back a mix of mSOL and Liquidity Pool tokens to Marinade.
 
-While the protocol is in development, yield remains in the protocol itself. 
+                                                                    
+## Sunrise Stake treasury spending 
 
-In the future, yield will be directed to purchasing carbon offset tokens, which will be
-subsequently burned in order to retire the underlying carbon credits.
+Thanks to the yield earned in the staking process, Sunrise can purchase carbon tokens and burn them to offset carbon emissions. 
 
-Long-term, the decision of where to direct yield will be made by the community, governed by
-the [Sunrise Stake DAO](https://app.realms.today/realm/SunriseStake).
+#### 1. Yield transfer to the treasury account
 
-## Governance
+The yield earned from staking SOL is transferred to the __treasury account__. It is a Program Derived Address ([PDA](https://docs.solana.com/developing/programming-model/calling-between-programs#program-derived-addresses)) meaning that it is owned and controlled by a smart contract, not a private key. This smart contract is known as the __treasury controller__. It can trigger a state change to buy or burn carbon tokens, thus retiring underlying carbon credits.
```

Comment:
Some duplication here with the above. IMO the explanation of the treasury management is better suited down here.